portal/models/questionnaire_bank.py

```diff
@@ -462,3 +463,21 @@ def add_if_not_found(self, commit_immediately=False):
             self.id = existing.id
         self = db.session.merge(self)
         return self
+
+
+def visit_name(qbd):
+    if not qbd.questionnaire_bank:
+        return None
+    name = sub('_', ' ', qbd.questionnaire_bank.name).split()[0]
```

Comment:
string.replace should cut it here - quicker and safer than re, unless you need pattern matching.